### PR TITLE
#595 Summary page styles

### PR DESCRIPTION
**Pull request by CarlosNZ**
Implements basic layout and styles for Summary page.

Note, this is only done for applicant's FIRST application, the styles for Changes requested and re-submissions are not included here (as I think they should be done after basic Review styles)

Please check for both before and after submission of application.

### CODE REVIEW

File: src/containers/Application/ApplicationSummary.tsx [hunk 1]
```diff
@@ -23,6 +24,8 @@ const ApplicationSummary: React.FC<ApplicationProps> = ({
 
   usePageTitle(strings.PAGE_TITLE_APPLICATION.replace('%1', fullStructure.info.serial))
 
+  console.log('fullStructure', fullStructure)
```
Comment (andreievg) [inline src/containers/Application/ApplicationSummary.tsx:27]: stray log

File: src/components/PageElements/PageElements.tsx [hunk 2]
```diff
@@ -106,7 +106,11 @@ const PageElements: React.FC<PageElementProps> = ({
 
           return (
             <div key={`question_${element.id}`}>
-              <Segment style={inlineStyles(isChangeRequest, isSummary)}>
+              <Segment
+                basic
+                className="summary-page-element"
+                style={inlineStyles(isChangeRequest, isSummary)}
```
Comment (andreievg) [inline src/components/PageElements/PageElements.tsx:112]: I guess you'll remove that in the change request branch

**Review by andreievg — CHANGES_REQUESTED** [review]
A couple of tiny changes, looks good thanks

File: semantic/src/site/globals/site.overrides [hunk 3]
```diff
@@ -478,6 +478,70 @@ a:hover {
...
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.summary-section {
+  background-color: @surfacesLight;
+  border-radius: @borderRadius;
+  padding: 20px;
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply semantic/src/site/globals/site.overrides]: Okay, I did `padding: 15px 20px;` - I've been trying to keep paddings and margins to 5px units (except in occasional circumstances), as it generally keeps things simpler.

03692ac5

File: src/components/Sections/SectionWrapper.tsx [hunk 4]
```diff
@@ -36,20 +36,20 @@ const SectionWrapper: React.FC<SectionProps> = ({
...
-              <Grid.Column floated="right" textAlign="right">
+              <Grid.Column textAlign="right">
                 {extraSectionTitleContent && extraSectionTitleContent(section)}
               </Grid.Column>
-              <Grid.Column floated="right" textAlign="right" width={1}>
+              <Grid.Column textAlign="right" width={1}>
                 <Icon
-                  style={sectionStyles.icon}
                   name={isActive ? 'angle up' : 'angle down'}
+                  className="dark-grey"
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply src/components/Sections/SectionWrapper.tsx:57]: Cool, that's nicer. I went through and replaced them all :) adff1b6c

File: src/containers/Application/ApplicationSummary.tsx [hunk 5]
```diff
@@ -71,89 +74,47 @@ const ApplicationSummary: React.FC<ApplicationProps> = ({
...
-              color="blue"
+              primary
+              className="button-wide"
               onClick={handleSubmit}
-              content={strings.BUTTON_SUBMIT}
+              content={strings.BUTTON_SUBMIT_APPLICATION}
             />
+            <p>
+              <Link to={`/application/${fullStructure.info.serial}`}>
+                <strong>Back to form</strong>
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply src/containers/Application/ApplicationSummary.tsx]: 👍 36aae5fe

File: src/containers/Application/ApplicationSummary.tsx [hunk 6]
```diff
@@ -71,89 +74,47 @@ const ApplicationSummary: React.FC<ApplicationProps> = ({
...
-              style={inlineStyles.button}
-              color="blue"
+              primary
+              className="button-wide"
               onClick={handleSubmit}
-              content={strings.BUTTON_SUBMIT}
+              content={strings.BUTTON_SUBMIT_APPLICATION}
             />
+            <p>
+              <Link to={`/application/${fullStructure.info.serial}`}>
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply src/containers/Application/ApplicationSummary.tsx:109]: I dunno, it says "Introduction", so it probably should go the "Intro"/"Home" page (there's already a link to the first page).

If we don't like this, maybe we just remove it or change it to something else?

File: src/containers/Application/ApplicationSummary.tsx (hunk 1 above)
Comment (CarlosNZ) [reply src/containers/Application/ApplicationSummary.tsx]: 2f57dc36

File: src/components/PageElements/PageElements.tsx (hunk 2 above)
Comment (CarlosNZ) [reply src/components/PageElements/PageElements.tsx:112]: Yeah, that's the plan. I think that these dynamic borders might need to be inline styles -- it seems like the simplest way, but we can address that when we get to these.

**Review by andreievg — APPROVED** [review]
Looks good thanks

**Pull request merged**